test/test-ghe-backup-config.sh

```diff
@@ -127,16 +127,3 @@ begin_test "ghe-backup-config ghe_parse_remote_version v2.x series"
 )
 end_test
 
-begin_test "ghe-backup-config verbose log redirects to file"
-(
-  set -e
-
-  export GHE_VERBOSE=1
-  export GHE_VERBOSE_LOG="$TRASHDIR/verbose.log"
-  . "share/github-backup-utils/ghe-backup-config"
-  ghe_verbose "Hello world"
-  [ "$(wc -l <"$GHE_VERBOSE_LOG")" -gt 0 ]
-  unset GHE_VERBOSE
-  unset GHE_VERBOSE_LOG
-)
-end_test
```

Comment:
Removing this test because of buffering issues surrounding fd 3 redirection. The log will eventually get written to, but not in time for this test to successfully pass.

Reply:
This delay can't be very long. Wouldn't a `sleep 1` or a `while` loop be a better option than risking the chance of breaking verbose logging without realising it and only discovering when a critical issue needs debugging?

I'll leave that judgment to you and not block this PR for this.

Reply:
I was able to fix this bug by leveraging `awk`'s `fflush()`
```
exec 3> >(awk -v c=$calling_script_name '{ print strftime("%b %d %H:%M:%S"), c":", $0; fflush(); }' >> "$GHE_VERBOSE_LOG")
```

I've added the test back in and it's passing.